app/domain/authentication/authn/authenticator.rb

```diff
@@ -1,40 +1,54 @@
 # frozen_string_literal: true
 
+require 'command_class'
```

Comment:
General comment: Please clean up the history before merging.

Reply:
Not sure I follow.

Reply:
your git history needs cleaning.